src/main/java/com/thed/zephyr/jenkins/reporter/ZeeDescriptor.java

```diff
@@ -390,17 +393,12 @@ public ListBoxModel doFillCycleDurationItems(
 		return listBoxModel;
 	}
 
-    public ListBoxModel doFillParserIndexItems() {
+    public ListBoxModel doFillParserTemplateKeyItems(@QueryParameter String parserTemplateKey) throws URISyntaxException {
```

Comment:
How this method gets called doFillParserTemplateKeyItems?

Reply:
Jenkins will call by itself when config screen is opened.